the fod vendor hash isn't reproducible since the switch to go 1.18:
```
 nix-build -A vgo2nix.go-modules --check
this path will be fetched (13.22 MiB download, 14.71 MiB unpacked):
  /nix/store/njryw1dchsnkvrmj8h0ddpsfzn4c9r26-vgo2nix-unstable-2020-11-07-go-modules
copying path '/nix/store/njryw1dchsnkvrmj8h0ddpsfzn4c9r26-vgo2nix-unstable-2020-11-07-go-modules' from 'https://cache.nixos.org'...
checking outputs of '/nix/store/x45m3y3g87gpcqpylshl4cppc04kffkr-vgo2nix-unstable-2020-11-07-go-modules.drv'...
unpacking sources
unpacking source archive /nix/store/ijk04rpm1j49015rp554p6xp48jxmhck-source
source root is source
patching sources
configuring
building
installing
error: hash mismatch in fixed-output derivation '/nix/store/x45m3y3g87gpcqpylshl4cppc04kffkr-vgo2nix-unstable-2020-11-07-go-modules.drv':
         specified: sha256-NvF22R/ZubTB2kE0BzlTMGtPHm+EdnUx/nRlgLci9PU=
            got:    sha256-206TT+52RQLkEK4CLOy2YvAWKD+FH9pv+Xk2bRBg34w=
```
ppdating the vendor hash causes the error.
```
nix-build -A vgo2nix
this derivation will be built:
  /nix/store/l5h74gr765kzzy4hz2j886j28xgjqfz8-vgo2nix-unstable-2020-11-07.drv
building '/nix/store/l5h74gr765kzzy4hz2j886j28xgjqfz8-vgo2nix-unstable-2020-11-07.drv'...
unpacking sources
unpacking source archive /nix/store/ijk04rpm1j49015rp554p6xp48jxmhck-source
source root is source
patching sources
configuring
building
Building subPackage ./.
go: downloading golang.org/x/mod v0.3.0
go: downloading github.com/orivej/go-nix v0.0.0-20180830055821-dae45d921a44
go: downloading golang.org/x/tools v0.0.0-20200904185747-39188db58858
go: downloading github.com/orivej/e v0.0.0-20180728214217-ac3492690fda
go: downloading golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1
../go/pkg/mod/github.com/orivej/go-nix@v0.0.0-20180830055821-dae45d921a44/nix/eval/eval.go:8:2: github.com/orivej/e@v0.0.0-20180728214217-ac3492690fda: reading file:///nix/store/7666gs0nymxlv2rfb998fc53wpn761sl-vgo2nix-unstable-2020-11-07-go-modules/github.com/orivej/e/@v/v0.0.0-20180728214217-ac3492690fda.zip: no such file or directory
../go/pkg/mod/golang.org/x/mod@v0.3.0/module/module.go:106:2: golang.org/x/xerrors@v0.0.0-20200804184101-5ec99f83aff1: reading file:///nix/store/7666gs0nymxlv2rfb998fc53wpn761sl-vgo2nix-unstable-2020-11-07-go-modules/golang.org/x/xerrors/@v/v0.0.0-20200804184101-5ec99f83aff1.zip: no such file or directory
error: builder for '/nix/store/l5h74gr765kzzy4hz2j886j28xgjqfz8-vgo2nix-unstable-2020-11-07.drv' failed with exit code 1;
       last 10 log lines:
       > configuring
       > building
       > Building subPackage ./.
       > go: downloading golang.org/x/mod v0.3.0
       > go: downloading github.com/orivej/go-nix v0.0.0-20180830055821-dae45d921a44
       > go: downloading golang.org/x/tools v0.0.0-20200904185747-39188db58858
       > go: downloading github.com/orivej/e v0.0.0-20180728214217-ac3492690fda
       > go: downloading golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1
       > ../go/pkg/mod/github.com/orivej/go-nix@v0.0.0-20180830055821-dae45d921a44/nix/eval/eval.go:8:2: github.com/orivej/e@v0.0.0-20180728214217-ac3492690fda: reading file:///nix/store/7666gs0nymxlv2rfb998fc53wpn761sl-vgo2nix-unstable-2020-11-07-go-modules/github.com/orivej/e/@v/v0.0.0-20180728214217-ac3492690fda.zip: no such file or directory
       > ../go/pkg/mod/golang.org/x/mod@v0.3.0/module/module.go:106:2: golang.org/x/xerrors@v0.0.0-20200804184101-5ec99f83aff1: reading file:///nix/store/7666gs0nymxlv2rfb998fc53wpn761sl-vgo2nix-unstable-2020-11-07-go-modules/golang.org/x/xerrors/@v/v0.0.0-20200804184101-5ec99f83aff1.zip: no such file or directory
       For full logs, run 'nix log /nix/store/l5h74gr765kzzy4hz2j886j28xgjqfz8-vgo2nix-unstable-2020-11-07.drv'.
```

the package probably should have been pinned to go 1.17 with the comment used elsewhere:

`# pinned due to build failure or vendoring problems. When unpinning double check with: nix-build -A $name.go-modules --rebuild`

--- a/pkgs/development/tools/vgo2nix/default.nix
+++ b/pkgs/development/tools/vgo2nix/default.nix
@@ -17,8 +17,7 @@ buildGoModule {
     sha256 = "0n9pf0i5y59kiiv6dq8h8w1plaz9w6s67rqr2acqgxa45iq36mkh";
   };
 
-  vendorSha256 = "1xgl4avq0rblzqqpaxl4dwg4ysrhacwhfd21vb0v9ffr3zcpdw9n";
-  proxyVendor = true;
+  vendorSha256 = "sha256-e5n0QhgffzC1igZX1zS18IX5oF5F0Zy/s8jWyNOD8NM=";
 
   subPackages = [ "." ];
 
@@ -36,8 +35,5 @@ buildGoModule {
     license = licenses.mit;
     maintainers = with maintainers; [ adisbladis ];
     mainProgram = "vgo2nix";
-    # vendoring fails with cryptic error:
-    # reading file:///nix/store/..../github.com/orivej/e/@v/v0.0.0-20180728214217-ac3492690fda.zip: no such file or directory
-    broken = true;
   };
 }

